lib/hammer_cli_foreman_tasks/task_progress.rb

```diff
@@ -61,11 +62,15 @@ def progress_bar
       bar.settings.tty.finite.template.main    = '[${<bar>}] [${<percent>%}]'
       bar.settings.tty.finite.template.padchar = ' '
       bar.settings.tty.finite.template.barchar = '.'
-      bar.settings.tty.finite.output           = Proc.new { |s| $stderr.print s }
+      bar.settings.tty.finite.output           = Proc.new { |s| $stderr.print s if appropriate_verbosity? }
       yield bar
     ensure
       bar.close
       render_result
     end
+
+    def appropriate_verbosity?
```

Comment:
Without looking where it is used in the code it is not really obvious for what is the verbosity appropriate

Reply:
I guess I forgot to add here a `private` access modifier, because it's not meant to be a public method.

Or you mean I should rename it properly as well?